src/scheduler/analysis.py

```diff
@@ -396,8 +385,9 @@ def _get_blacklist_and_whitelist(self, next_analysis: str) -> Tuple[List, List]:
         return blacklist, whitelist
 
     def _get_blacklist_and_whitelist_from_config(self, analysis_plugin: str) -> Tuple[List, List]:
-        blacklist = read_list_from_config(self.config, analysis_plugin, 'mime_blacklist')
-        whitelist = read_list_from_config(self.config, analysis_plugin, 'mime_whitelist')
+
+        blacklist = parse_comma_separated_list(getattr(cfg, analysis_plugin, {}).get('mime_blacklist', ''))
+        whitelist = parse_comma_separated_list(getattr(cfg, analysis_plugin, {}).get('mime_whitelist', ''))
```

Comment:
See the other comment.

Still this one is a bit different.
I assume that every plugin may have a `mime_{white,black}list` in the config, does it?
If so we could do this in `config.load`.

Reply:
> I assume that every plugin may have a mime_{white,black}list in the config, does it?
If so we could do this in config.load.

Yes that would also make more sense IMHO (and it could also catch possible syntax errors in the config earlier)

Reply:
Done.